src/dmd/backend/elfobj.c

```diff
@@ -2764,86 +2834,84 @@ void Obj::addrel(int seg, targ_size_t offset, unsigned type,
     }
 }
 
-static size_t relsize64(unsigned type)
+private size_t relsize64(uint type)
 {
     assert(I64);
     switch (type)
     {
-    case R_X86_64_NONE: return 0;
-    case R_X86_64_64: return 8;
-    case R_X86_64_PC32: return 4;
-    case R_X86_64_GOT32: return 4;
-    case R_X86_64_PLT32: return 4;
-    case R_X86_64_COPY: return 0;
-    case R_X86_64_GLOB_DAT: return 8;
-    case R_X86_64_JUMP_SLOT: return 8;
-    case R_X86_64_RELATIVE: return 8;
-    case R_X86_64_GOTPCREL: return 4;
-    case R_X86_64_32: return 4;
-    case R_X86_64_32S: return 4;
-    case R_X86_64_16: return 2;
-    case R_X86_64_PC16: return 2;
-    case R_X86_64_8: return 1;
-    case R_X86_64_PC8: return 1;
-    case R_X86_64_DTPMOD64: return 8;
-    case R_X86_64_DTPOFF64: return 8;
-    case R_X86_64_TPOFF64: return 8;
-    case R_X86_64_TLSGD: return 4;
-    case R_X86_64_TLSLD: return 4;
-    case R_X86_64_DTPOFF32: return 4;
-    case R_X86_64_GOTTPOFF: return 4;
-    case R_X86_64_TPOFF32: return 4;
-    case R_X86_64_PC64: return 8;
-    case R_X86_64_GOTOFF64: return 8;
-    case R_X86_64_GOTPC32: return 4;
-    default:
-        break;
+        case R_X86_64_NONE:      return 0;
+        case R_X86_64_64:        return 8;
+        case R_X86_64_PC32:      return 4;
+        case R_X86_64_GOT32:     return 4;
+        case R_X86_64_PLT32:     return 4;
+        case R_X86_64_COPY:      return 0;
+        case R_X86_64_GLOB_DAT:  return 8;
+        case R_X86_64_JUMP_SLOT: return 8;
+        case R_X86_64_RELATIVE:  return 8;
+        case R_X86_64_GOTPCREL:  return 4;
+        case R_X86_64_32:        return 4;
+        case R_X86_64_32S:       return 4;
+        case R_X86_64_16:        return 2;
+        case R_X86_64_PC16:      return 2;
+        case R_X86_64_8:         return 1;
+        case R_X86_64_PC8:       return 1;
+        case R_X86_64_DTPMOD64:  return 8;
+        case R_X86_64_DTPOFF64:  return 8;
+        case R_X86_64_TPOFF64:   return 8;
+        case R_X86_64_TLSGD:     return 4;
+        case R_X86_64_TLSLD:     return 4;
+        case R_X86_64_DTPOFF32:  return 4;
+        case R_X86_64_GOTTPOFF:  return 4;
+        case R_X86_64_TPOFF32:   return 4;
+        case R_X86_64_PC64:      return 8;
+        case R_X86_64_GOTOFF64:  return 8;
+        case R_X86_64_GOTPC32:   return 4;
```

Comment:
I hate these alignments. Because if an additional case is added which are longer than the existing ones, they all need to be realigned, which is a waste of time.

Reply:
The original was unreadable.